2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_setup.py:_flush():69] setting env: {}
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_init.py:_log_setup():336] Logging user logs to /Users/master/Documents/School/CSC498/finalProject/CartPoleExperiment/wandb/run-20210416_161902-f9raw1g2/logs/debug.log
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/master/Documents/School/CSC498/finalProject/CartPoleExperiment/wandb/run-20210416_161902-f9raw1g2/logs/debug-internal.log
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_init.py:init():369] calling init triggers
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-16 16:19:02,387 INFO    MainThread:12719 [wandb_init.py:init():418] starting backend
2021-04-16 16:19:02,388 INFO    MainThread:12719 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-04-16 16:19:02,403 INFO    MainThread:12719 [backend.py:ensure_launched():123] starting backend process...
2021-04-16 16:19:02,409 INFO    MainThread:12719 [backend.py:ensure_launched():127] started backend process with pid: 12725
2021-04-16 16:19:02,412 INFO    MainThread:12719 [wandb_init.py:init():423] backend started and connected
2021-04-16 16:19:02,413 INFO    MainThread:12719 [wandb_init.py:init():465] updated telemetry
2021-04-16 16:19:02,413 INFO    MainThread:12719 [wandb_init.py:init():484] communicating current version
2021-04-16 16:19:03,407 INFO    MainThread:12719 [wandb_init.py:init():489] got version response 
2021-04-16 16:19:03,407 INFO    MainThread:12719 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-16 16:19:03,537 INFO    MainThread:12719 [wandb_init.py:init():522] starting run threads in backend
2021-04-16 16:19:03,558 INFO    MainThread:12719 [wandb_run.py:_console_start():1524] atexit reg
2021-04-16 16:19:03,559 INFO    MainThread:12719 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-16 16:19:03,559 INFO    MainThread:12719 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-16 16:19:03,560 INFO    MainThread:12719 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-16 16:19:03,560 INFO    MainThread:12719 [wandb_init.py:init():546] run started, returning control to user process
2021-04-16 16:19:05,401 INFO    MainThread:12719 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-16 16:19:05,402 INFO    MainThread:12719 [wandb_run.py:_restore():1466] restore
2021-04-16 16:19:07,544 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 499
  total_bytes: 499
}

2021-04-16 16:19:08,562 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 499
  total_bytes: 7664
}

2021-04-16 16:19:08,699 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 499
  total_bytes: 7666
}

2021-04-16 16:19:08,802 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 499
  total_bytes: 7666
}

2021-04-16 16:19:08,907 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7666
  total_bytes: 7666
}

2021-04-16 16:19:09,015 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7666
  total_bytes: 7666
}

2021-04-16 16:19:09,121 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7666
  total_bytes: 7666
}

2021-04-16 16:19:09,226 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7666
  total_bytes: 7666
}

2021-04-16 16:19:09,332 INFO    MainThread:12719 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7666
  total_bytes: 7666
}

2021-04-16 16:19:10,439 INFO    MainThread:12719 [wandb_run.py:_show_files():1838] logging synced files
